uoyweek.cpp

```diff
@@ -19,7 +34,8 @@ class Term {
         code = c;
         startTS = s;
         endTS = e;
-        //Store actual value instead of reference to gmtime so as to avoid issues
+        /* Store actual value instead of reference to gmtime so as to avoid
+         * issues */
```

Comment:
something something 80 chars wrap. This is why I prefer 120 chars, or at least trying to match the line lengths